istioctl/pkg/authz/listener.go

```diff
@@ -156,7 +156,7 @@ func Print(writer io.Writer, listeners []*listener.Listener) {
 					addPolicy(action, nameOfPolicy, indexOfRule)
 				}
 				if len(rbacHTTP.GetRules().GetPolicies()) == 0 {
-					addPolicy(action, anonymousName, "0")
+					addPolicy(rbacpb.RBAC_DENY, denyAll, "0")
```

Comment:
Doesn't it depend on the action? This could be `RBAC_ALLOW` action in which case its really "allow all"?